rskj-core/src/main/resources/config/rsk-sample.conf

```diff
@@ -151,20 +151,6 @@ sync {
     peer.count = 10
 }
 
-# miner options
```

Comment:
Removing this from sample config file is fine but the [wiki page](https://github.com/rsksmart/rskj/wiki/Configure-your-RSK-Miner-Node) that indicates how to setup a miner node is not clear now. 
Reason is that there are two sections where mining settings are set (https://github.com/rsksmart/rskj/wiki/Configure-your-RSK-Miner-Node and https://github.com/rsksmart/rskj/wiki/Configure-your-RSK-Miner-Node). I suggest merging them somehow. We can review this issue together doing pair documenting :)

Reply:
That would be awesome :grin:

Reply:
Let's do this tomorrow!

Reply:
Done!